packages/cpsuLambda/src/schema/format_1/transformer.ts

```diff
@@ -10,13 +10,16 @@ import {v4 as uuidv4} from "uuid"
 import {Transformer} from "../../handler"
 import {wrap_with_status} from "../../utils"
 import {Md5} from "ts-md5"
+import {Logger} from "@aws-lambda-powertools/logger"
+
+const logger = new Logger({serviceName: "CustomUpdatePrescriptionStatus"})
```

Comment:
We should use the _logger passed into the transformer function instead of creating this new one. There's an append_headers function that makes sure it has all the required headers for tracking and it'll have the correct LOG_LEVEL set from the lambda env vars.

Reply:
Good catch - thanks